package-lock.json

```diff
@@ -6,13 +6,13 @@
   "dependencies": {
     "acorn": {
       "version": "5.1.2",
-      "resolved": "https://registry.npmjs.org/acorn/-/acorn-5.1.2.tgz",
+      "resolved": "https://repo.bazaarvoice.com/nexus/content/groups/npm-all/acorn/-/acorn-5.1.2.tgz",
```

Comment:
I'm not sure what we are supposed to do with this package.lock file (I'm not an npm expert), it looks like a bunch of the same changes were generated. Is this supposed to be committed? Any ideas?

Reply:
Oops! Yeah I'm not clear on how to handle `package-lock.json` either - I think it was introduced in NPM 5.

I can remove it (I do this in my projects) but when I rebased it was already in source control so you may run into issues with it again.